src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -1841,7 +1841,7 @@ subroutine RUN ( GC, IMPORT, EXPORT, CLOCK, RC )
          trim(out_path), trim(exp_id), exp_domain,                         &
          met_force, lai, cat_param, mwRTM_param,                           &
          tile_coord_l, tile_coord_rf,                                      &
-         tcinternal%pgrid_f, tcinternal%pgrid_g, tcinternal%grid_g,        &
+         tcinternal%tgrid_g, tcinternal%pgrid_f, tcinternal%pgrid_g,       &
```

Comment:
@weiyuan-jiang: I changed the order of the arguments here -- it's more intuitive to have "tile_grid" before "pert_grid", at least to me.